server/index.js

```diff
@@ -2,19 +2,20 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const Promise = require('bluebird');
 const db = require('../database/index.js');
-
-
 const app = express();
 
-
+app.use(express.static(__dirname + '/../client/dist'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-app.use(express.static(__dirname + '/../client/dist'));
 
+app.get('/', (req, res) => {
+  console.log("afdghadgf")

```

Comment:
You may want to write more specific logs - these can become very easy to lose track of!